src/plugins/chart_expressions/expression_xy/common/types/expression_functions.ts

```diff
@@ -248,6 +252,8 @@ export interface XYProps {
   hideEndzones?: boolean;
   valuesInLegend?: boolean;
   ariaLabel?: string;
+  splitRowAccessor?: ExpressionValueVisDimension | string;
```

Comment:
Not a strong opinion, but putting these on the chart level implies that the split columns need to have the same id across all layers. Does this make sense? Or should we put it on the individual layer instead to allow for different split accessors per layer? I think it would be a little more complex to support this as part of the split accessor function but it should be possible.

Reply:
You know, I was playing around with the `splitRow` and `splitColumn` functionality and came to the idea, that it is not supported by the `elastic-charts` for now (multiple `splitRows` and `splitColumns` per one chart). So, I've implemented it only for the xyVis. Only one `SmallMultiples` component is supported. Does it make sense?